src/vivarium_csu_ltbi/components/disease.py

```diff
@@ -11,7 +18,193 @@ def wrap_data_getter(id):
     }
 
 
-class BetterDiseaseModel(DiseaseModel):
+class BetterDiseaseModel(Machine):
+    """FIXME: This class used to extend DiseaseModel itself, but the population
```

Comment:
direct copy of DiseaseModel + additional value dependencies on the population initializer

Reply:
Let's take 15 mins monday and talk about how to incorporate this fix into vph.